### PR TITLE
Updated docker-compose and Dockerfile

### PR DESCRIPTION
**Pull request by ExpressHermes**
**What have you Changed**

- Updated Dockerfile and docker-compose.yaml to run the app properly
- Added error handling in case the required env variables are not provided

### Self Check (Tick After Making pull Request)

- [x] One change in one Pull Request
- [x] I am following clean code and Documentation and my code is well linted with flake8.

Join Us on Discord:- https://discord.gg/JfbK3bS

## acknowledgement

- I have read and understood the above instructions. Your pull request will be counted as an open-source pull request and will be distributed under the MIT license.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,9 +2,9 @@ FROM python:3.9
...
 
-COPY requirements /app/requirements
+COPY requirements.txt /app/requirements.txt
 
 RUN pip install -r requirements.txt
 COPY . /app
 
-RUN start.sh
\ No newline at end of file
+CMD bahs start_server.sh
```
Comment (codeperfectplus) [inline Dockerfile]: Spelling error for the bash. 

```
RUN bash start_server.sh
```
```
CMD bahs start_server.sh
```

I am not sure about cmd, cmd is for windows only commands right? will it work on Docker?

File: README.md [hunk 2]
```diff
@@ -65,14 +65,27 @@ Invite **Cybel** into your server
...
+```bash
+DISCORD_TOKEN=Discord bot token [get it from here](https://discord.com/developers/applications)
+WEATHER_API_KEY=OpenWeatherMap API key [get it from here](https://openweathermap.org/api)
+DATABASE_URL=postgres://username:password@localhost:5432/database_name
+```
+
+2. Run the docker-compose command to run the bot
+
+```bash
+docker compose up
```
Comment (codeperfectplus) [inline README.md]: docker compose up -d
 
 to run in reattached mode

**Review by codeperfectplus — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ExpressHermes) [reply Dockerfile]: Done

File: README.md (hunk 2 above)
Comment (ExpressHermes) [reply README.md]: Done

**Review by codeperfectplus — APPROVED** [review]

**Pull request merged**